client/apis/volumegroupsnapshot/v1alpha1/types.go

```diff
@@ -354,10 +354,27 @@ type VolumeGroupSnapshotContentSource struct {
 	// +optional
 	PersistentVolumeNames []string `json:"persistentVolumeNames,omitempty" protobuf:"bytes,1,opt,name=persistentVolumeNames"`
 
+	// GroupSnapshotHandleSource specifies the CSI "group_snapshot_id" of a pre-existing
+	// group snapshot and a list of CSI "snapshot_id" of pre-existing snapshots
+	// on the underlying storage system for which a Kubernetes object
+	// representation was (or should be) created.
+	// This field is immutable.
+	// +optional
+	GroupSnapshotHandleSource *VolumeGroupSnapshotHandleSource `json:"groupSnapshotHandleSource,omitempty" protobuf:"bytes,2,opt,name=groupSnapshotHandleSource"`
```

Comment:
I would call this "GroupSnapshotSource" since this is more than just a handle.

Reply:
We've already have "VolumeGroupSnapshotSource" inside "VolumeGroupSnapshotSpec":
https://github.com/kubernetes-csi/external-snapshotter/pull/971/files#diff-b1276d82fb92834c7dadee3a748b9b88488fecb250d9add1a1dc1d5618a5c1cbL47

Reply:
Discussed with Michelle.  We'll rename to `GroupSnapshotHandles *GroupSnapshotHandles` to avoid having conflicts with the existing names.

Another thing is we should try to be consistent with the design for VolumeSnapshot APIs. We have "VolumeHandle *string" in VolumeSnapshotContentSource. So we should check if we can have `VolumeHandles []string` here instead of `PersistentVolumeNames []string`.
https://github.com/kubernetes-csi/external-snapshotter/blob/v6.3.3/client/apis/volumesnapshot/v1/types.go#L354

Reply:
@xing-yang @msau42 thanks for the reviews! 
I've updated the APIs in the latest commit. The API name changes also required some changes to the controller logic, so I've added that too. PTAL!